crates/anvil/src/eth/api.rs

```diff
@@ -727,6 +730,36 @@ impl EthApi {
         self.backend.get_account_at_block(address, Some(block_request)).await
     }
 
+    /// Returns the account information including balance, nonce, code and storage
+    pub async fn get_account_info(
+        &self,
+        address: Address,
+        block_number: Option<BlockId>,
+    ) -> Result<serde_json::Value> {
+        node_info!("eth_getAccountInfo");
+        let block_request = self.block_request(block_number).await?;
+
+        if let BlockRequest::Number(number) = block_request {
+            if let Some(fork) = self.get_fork() {
+                if fork.predates_fork(number) {
+                    let acc = fork.get_account(address, number).await?;
+                    return Ok(serde_json::json!({
+                        "balance": acc.balance,
+                        "nonce": acc.nonce,
+                        "code": acc.code_hash
+                    }));
+                }
+            }
+        }
```

Comment:
we can skip this part part for this request because we dont expect that this is supported by the fork client